sdk/src/serializable.ts

```diff
@@ -35,7 +35,7 @@ export interface TypedJson {
 export type Serializable =
   | protobuf.Message
   | TypedJson
-  | any
+  | object
```

Comment:
This change means that `Serializable` is no longer always just `any`. The handlers and callbacks can't use the Serialiable union type directly now, as it would require the implementation to accept all thse types. Entities and their handlers and contexts will also become generic, with a parameter that extends Serializable instead.